app/models/concerns/relatable.rb

```diff
@@ -28,6 +28,8 @@ def self.get_related_model(counter_name)
       counter_name = counter_name.split(/ as /i)[-1]  # honor aliasing
       counter_name.gsub!('_count','')   # support cached count columns as well
 
+      counter_name.gsub!('qtls','qtl')  # unfortunate special case... :(
```

Comment:
The "l" in `qtl` stands for "locus", and the plural of "locus" is "loci", so `qtls` is just bad grammar.